packages/lib/response/util.ts

```diff
@@ -971,6 +981,58 @@ export const getQuestionWiseSummary = (
 
         break;
       }
+      case TSurveyQuestionType.Matrix: {
+        const rows = question.rows;
+        const columns = question.columns;
+        let totalResponseCount = 0;
+
+        // Initialize count object
+        const countMap: Record<string, string> = rows.reduce((acc, row) => {
+          acc[row] = columns.reduce((colAcc, col) => {
+            colAcc[col] = 0;
+            return colAcc;
+          }, {});
+          return acc;
+        }, {});
```

Comment:
Can we have better naming variables for more readability here?

Reply:
We already use choiceCountMap in multiChoice summary so to keep it consistent, its named this way and also its a map of counting and hence is being named countMap 😊